tests/functional/intelligent-host-selection/05-from-platform-group.t

```diff
@@ -39,14 +43,24 @@ create_test_global_config "" "
 
 [platform groups]
     [[mixedplatformgroup]]
-        platforms = badhostplatform, ${CYLC_TEST_PLATFORM}
+        platforms = badhostplatform, mixedhostplatform
```

Comment:
This is the core problem: Users can change `${CYLC_TEST_PLATFORM}` wheras we want a group with fixed platforms in it.